File: Práctica 1/Java/practica1-prueba/p04.java
//ANTONIO JOSE FERNANDEZ BELLURE

//prueba04 de las del corrector de Alicia(supuestamente)

import java.io.*;
import java.util.ArrayList;

public class p04{

	public static void main(String[] args) {

		Coleccion colec = new Coleccion();

		colec.lectura(args[0]);//este fichero solo tiene el mapa

		Localidad localidad = new Localidad("Fnatic City");
		InfoTur info = new InfoTur(2,5,1,2,true);
		String top="*** e-Sports Reckless stadium";
		info.setTop(top);

		localidad.setInfo(info);
		localidad.setCoor(2,5,colec.getMapa());

		System.out.println(localidad);
		System.out.println(colec);

		colec.getLocalidades().add(localidad);

		System.out.println(colec);
	}
}

File: Práctica 1/C++/practica1-prueba/p04.cc
#include <iostream>
#include <vector>
#include <fstream>
#include <sstream>
#include <cstdlib>
#include "Coleccion.h"

using namespace std;

int main(int argc, char *argv[]){

	Coleccion colec = Coleccion();
	string fichero=argv[1];
	colec.lectura(fichero);

	Localidad localidad=Localidad("Fnatic City");
	InfoTur info=InfoTur(2,5,1,2,true);
	string top="*** e-Sports Reckless stadium";
	info.setTop(top);

	localidad.setInfo(info);
	localidad.setCoor(2,5,colec.getMapa());

	cout<<localidad<<endl;
	cout<<colec<<endl;

	colec.getLocalidades().push_back(localidad);

	cout<<colec<<endl;

	return 0;
}